crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -80,6 +80,7 @@ pub struct InboundGroupSession {
     imported: bool,
     algorithm: Arc<EventEncryptionAlgorithm>,
     backed_up: Arc<AtomicBool>,
+    safety: KeySafety,
```

Comment:
I think I'd rather have the `trusted` or rather `claimed_trust` flag here instead of an enum that tells us about safety without a proof.

Reply:
I didn't want to use trust, as it's not related to verification. And we will later have something regarding trust.
Safety is not without a proof, it means that this megolm key was created by the sender (sender_key) of this message.
WDYM by proof?

Reply:
From out of band, I think the latter part of @poljar's comment was mostly aimed at MSC3879 rather than anything in this PR. The point being that the value of the MSC3879 `trusted` field is entirely predicated on us believing/trusting the sender rather than a cryptographic proof, so having a `claimed_`  prefix in the name might be prudent.

Reply:
done